src/main/kotlin/step3/CarFactory.kt

```diff
@@ -0,0 +1,13 @@
+package step3
+
+class CarFactory {
+    fun create(numberOfCars: Int): List<Car> {
+        val cars: MutableList<Car> = mutableListOf()
+
+        for (i in 1..numberOfCars) {
+            cars.add(Car())
+        }
+
+        return cars
```

Comment:
```suggestion
        return List(numberOfCars) { Car() }
```

다음처럼 작성하면 MutableList 생성 없이도 원하는 값을 얻을 수 있습니다. 코틀린의 Collection API엔 유용한 API가 많아요 (코틀린의 매력 중 하나 🙂)